### PR TITLE
Added numBytesToErase to CodeCompletionItem

### PR DESCRIPTION
**Pull request by KelvinJin**
This field is defined in the [protocol](https://github.com/apple/swift/blob/master/tools/SourceKit/docs/Protocol.md) and will be pretty useful to have in order to do autocompletion in an editor.

Tests passed locally on my mac. If it looks good, I'll update the change log too.

### CODE REVIEW

**Comment by KelvinJin** [comment]
hmm, seems tests failed under linux. Not sure why though.

File: Source/SourceKittenFramework/CodeCompletionItem.swift [hunk 1]
```diff
@@ -26,17 +26,19 @@ public struct CodeCompletionItem: CustomStringConvertible {
     public let moduleName: String?
     public let docBrief: String?
     public let associatedUSRs: String?
+    public let numBytesToErase: Int64?
```
Comment (norio-nomura) [inline Source/SourceKittenFramework/CodeCompletionItem.swift]: Changing this to `Int` will fix crash in `CodeCompletionTests` on Linux.

Comment (KelvinJin) [inline Source/SourceKittenFramework/CodeCompletionItem.swift]: Hmm, that's annoying. Have to add a typealias. I'm bad at naming.

**Comment by KelvinJin** [comment]
All right. Should be cool now.

**Comment by jpsim** [comment]
Thanks!

**Pull request merged**